### ISSUE TITLE
Mavros not starting

### ISSUE THREAD

**Issue by fnoop** [fnoop/maverick#655]
```
[apm.launch] is neither a launch file in package [mavros] nor is [mavros] a launch file name
The traceback for the exception was written to the log file
```

**Comment by fnoop** [comment]
ROS install flag gets installed even if non-core modules fail to build/install.

**Comment by fnoop** [comment]
mavros fixed for ubuntu 16.04 and raspbian.  Still getting merge conflicts for 17.04, pushing back to next release.
